docs/migration-guides/qiskit-backend-primitives.mdx

```diff
@@ -0,0 +1,79 @@
+the Primitive Unified Bloc (PUB) syntax, and the output formats should follow the
+V2 primitives standard. See the [Overview of PUBs section](/guides/primitive-input-output#pubs) in 
+the Primitive inputs and outputs guide for details.
+
+The two main differences with the implementations of [`SamplerV2`](/api/qiskit-ibm-runtime/sampler-v2) 
+and [`EstimatorV2`](/api/qiskit-ibm-runtime/estimator-v2) are:
+
+1.  The `backend` input passed during the "backend primitive" instantiation. This input argument allows to 
+internally wrap the access to the internal provider through `backend.run()` and it should not be confused
+with the `mode` input argument of [`SamplerV2`](/api/qiskit-ibm-runtime/sampler-v2) 
+and [`EstimatorV2`](/api/qiskit-ibm-runtime/estimator-v2).
+
+2. The use of `options`, which contains one more level of nesting compared to provider-specific primitives and
+is initially provided as a `dict` instead of an `Options` instance.
+In the first level of nesting, `BackendSamplerV2` includes `default_shots`, `seed_simulator` and `run_options`;
+`BackendEstimatorV2` includes `default_precision`, `abelian_grouping`, `seed_simulator` and `run_options`.
+The `run_options` field defines the options to pass through to the `.run()` method of the wrapped backend instance.
+
+Note that because these wrappers are independent of `qiskit_ibm_runtime`, they do not offer access to execution
+modes (`Session`/`batch`), local testing modes, or any other `qiskit_ibm_runtime`-specific feature. These are up to the
+third-party provider to implement and expose, ideally through a native primitive. 
+If the use of the third-party backend requires a more fine-grained option handling
+strategy that is not covered by the out-of-the-box wrapper, it is always possible to extend the 
+[`BackendSamplerV2`.](/api/qiskit/qiskit.primitives.BackendSamplerV2) and 
+[`BackendEstimatorV2`](/api/qiskit/qiskit.primitives.BackendEstimatorV2) interfaces through subclassing to fit the specific 
+interface needs.
+
+The following is an example usage of the [`BackendEstimatorV2`](/api/qiskit/qiskit.primitives.BackendEstimatorV2) 
+class with a fictional third-party provider backend that only implements `backend.run()`:
+
+``` python
+from qiskit import QuantumCircuit, generate_preset_pass_manager
+from qiskit.primitives import BackendSamplerV2
+from qiskit_fictional_provider import BackendRunProvider
+
+# get backend and pass onto BackendSamplerV2
+backend = BackendRunProvider("").get_backend("fictional_backend")
+sampler = BackendSamplerV2(backend, options={"default_shots": 1000})
+
+# from this point onwards, use sampler as any SamplerV2
+qc = QuantumCircuit(2)
+qc.cx(0, 1)
+qc.measure_all()
+
+# assuming that third-party backend is compatible with our transpilation pipeline
+# (whether this step is required or not will depend on the provider)
+pm = generate_preset_pass_manager(backend=backend)
+tqc = pm.run(qc)
+
+# run using pub syntax
+result = sampler.run(([qc])).result()
```

Comment:
The additional `()` doesn't do anything. If you want to make it explicitly a tuple, it needs to be `run(([qc],))`

```suggestion
result = sampler.run([qc]).result()
```